src/base/projects/Header.svelte

```diff
@@ -101,7 +101,7 @@
     on:branchChanged={event => updateRevision(event.detail)} />
 
   {#if seed.git.host}
-    <CloneButton seedHost={seed.git.host} {urn} />
+    <CloneButton {seed} {urn} />
```

Comment:
We should try to keep the interfaces of components lean.

That is, pass down only the required props and avoid forwarding "chunky" objects. This has the following benefits from my point of view:
  - we can see exactly which data goes into a component:
    - this makes reasoning about the surrounding code easier
    - this also adds friction to making components too complex, because you manually have to pass around props that you don't need, which in turn forces you to think more about what the component should actually do
    - reduces coupling and makes our components more composable
    - is potentially more resource efficient and performant
  - it makes it easier to test the component in isolation, because you don't have to set up complex objects

I think there are also exceptions to this rule, but generally I think we should stick to this. Wdyt @sebastinez?

Reply:
Yeah I agree with you on that.
In this case, the `CloneButton` was starting to need almost the entire `Seed` class, but in the end I reverted to only use the seed id, so I think I can move back to pass only the required props down.